Packs/CommonPlaybooks/ReleaseNotes/2_6_0.md

```diff
@@ -0,0 +1,16 @@
+
+#### Playbooks
+
+##### Calculate Severity By Highest DBotScore
+
+- The inputs of this playbook have changed. If you customized the playbook input to anything other than the DBotScore object, you will need to re-adjust the new inputs. The playbook now accepts a list of unique DBotScore indicators and the maximum score given to an indicator, instead of the full DBotScore object.
+- The performance of the playbook has been improved and the size of the context has been reduced.
+
+##### Calculate Severity - Generic v2
+
+- The inputs of this playbook have changed. If you customized the playbook input to anything other than the DBotScore object, you will need to re-adjust the new inputs. The playbook now accepts a list of unique DBotScore indicators and the maximum score given to an indicator, instead of the full DBotScore object.
+- The performance of the playbook has been improved and the size of the context has been reduced.
```

Comment:
```suggestion
- Changed the inputs of this playbook. If you customized the playbook input to anything other than the DBotScore object, you will need to re-adjust the new inputs. The playbook now accepts a list of unique DBotScore indicators and the maximum score given to an indicator, instead of the full DBotScore object.
- Improved the performance of the playbook and reduced the size of the context.
```